js/functions.js

```diff
@@ -630,4 +630,20 @@ function setupMobileTicker() {
 			startEmpty:true
 		});
 		
+}
+
+// Parses passed url and looks for the form 'https://:34343/something'. If after double slash after protocol there is no host present, a local host is assumed and is placed before port part.
+// The url from the example will be transformed into 'https://somehost:34343/something', where somehost is the host at which the muximux was accessed.
+function parseRelativeUrlWithPortIfPresent(url) {
+    var parts = /^(http|https|ftp):\/\/:(\d+)(.*)$/.exec(url),
```

Comment:
sure!
Does it handle for instance
https://username:password@:1234/someplace

if the user wants to input a username and password?